pages/download.vue

```diff
@@ -76,9 +83,23 @@
 
             ...mapState([
 
-                "dataTable"
+                "columnToCategoryMap",
+                "dataTable",
+                "missingValueLabel",
+                "toolGroups"
             ]),
 
+            datasetName() {
+
+                // Dataset name is original data table filename with no extension
+                return this.dataTable.filename.split(".").slice(0, -1).join(".");
+            },
+
+            defaultOutputFilename() {
+
+                return `${this.datasetName}_annotated_${Date.now()}.json`;
+            },
+
```

Comment:
Just wanted to say: very nice and clean stuff going on here! :tada:

Reply:
Danke!